src/OrchardCore.Modules/OrchardCore.Workflows/Views/Workflow/Index.cshtml

```diff
@@ -76,7 +76,7 @@
                             <div>
                                 <span class="hint">@T["Created {0}", (object)(await DisplayAsync(await New.TimeSpan(Utc: entry.Workflow.CreatedUtc)))]</span>
                                 <div class="info">
-                                    <span class="badge badge-@statusCss">@entry.Workflow.Status</span>
+                                    <span class="badge badge-@statusCss">@T[entry.Workflow.Status]</span>
```

Comment:
Enums should be localized using a switch statement that resolve to calls to T[] with the static string inside it. This way our tool will be able to pick it up and add the entry in pot files.